### PR TITLE
Fix for operator groups display when collapsing knative services

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes https://issues.redhat.com/browse/ODC-2850

/kind bug

### CODE REVIEW

File: frontend/packages/dev-console/src/components/topology/topology-utils.ts [hunk 1]
```diff
@@ -696,7 +696,7 @@ export const topologyModelFromDataModel = (
         type: d.type,
         label: dataModel.topology[d.id].name,
         data,
-        collapsed: filters && !filters.display.knativeServices,
+        collapsed: d.type === TYPE_KNATIVE_SERVICE && filters && !filters.display.knativeServices,
```
Comment (karthikjeeyar) [inline frontend/packages/dev-console/src/components/topology/topology-utils.ts]: @jeff-phillips-18 We need to support `TYPE_OPERATOR_BACKED_SERVICE` as well

Comment (jeff-phillips-18) [reply frontend/packages/dev-console/src/components/topology/topology-utils.ts]: Yes, that is story https://issues.redhat.com/browse/ODC-2851, not part of this PR.

**Pull request merged**